docker/server/Dockerfile

```diff
@@ -3,14 +3,14 @@ FROM ubuntu:16.04
 ARG repository="deb https://repo.yandex.ru/clickhouse/xenial/ dists/stable/main/binary-amd64/"
 ARG version=\*
 
-RUN apt-get update && \
-    apt-get install -y apt-transport-https tzdata && \
+RUN apt update && \
+    apt install -y apt-transport-https && \
     mkdir -p /etc/apt/sources.list.d && \
     echo $repository | tee /etc/apt/sources.list.d/clickhouse.list && \
-    apt-get update && \
-    apt-get install --allow-unauthenticated -y clickhouse-server-common=$version clickhouse-server-base=$version && \
+    apt update && \
+    apt install --allow-unauthenticated -y clickhouse-server-common=$version clickhouse-server-base=$version && \
```

Comment:
Is that change necessary? It breaks docker build since looks like apt ignores `--allow-unauthenticated` and fails to install clickhouse-server.
Since you're not changing 'FROM' in this file everything must be left as is.